src/lib/cxplat.user.vcxproj

```diff
@@ -1,71 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\cxplat.props" />
-  <ItemGroup>
-    <ClCompile Include="cxplat_winuser.c" />
-  </ItemGroup>
+  <PropertyGroup>
+    <ProjectGuid>{702A7493-A6A4-4209-AB69-E1E4D1EB0FB7}</ProjectGuid>
+    <TargetName>platformuser</TargetName>
+    <UndockedType>lib</UndockedType>
+    <UndockedDir>$(SolutionDir)submodules\undocked\</UndockedDir>
```

Comment:
when it's compiled in xdp, `SolutionDir` becomes the xdp solution folder.

Reply:
I don't know a good way to handle this. For this reason, I changed the path to cxplat.props to a relative path.